src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -54,7 +52,11 @@ public PesterCodeLensProvider(EditorSession editorSession)
                     new ClientCommand(
                         "PowerShell.RunPesterTests",
                         "Run tests",
-                        new object[] { scriptFile.ClientFilePath, false /* No debug */, pesterSymbol.TestName })),
+                        new object[] {
+                            scriptFile.DocumentUri,
+                            false /* No debug */,
+                            pesterSymbol.TestName,
+                            pesterSymbol.ScriptRegion?.StartLineNumber })),
```

Comment:
where did this change come from? Looks like it's from a different PR

Reply:
Indeed.  I could back this out but we do need this change pulled into v1.x.  But I imagine you'd prefer to merge this via the PR that impls the Pester invoke by line number enhancement?

Reply:
Yes I would :)